.github/workflows/rav1e.yml

```diff
@@ -283,15 +302,19 @@ jobs:
     - name: Set environment variables
       run: |
         echo "::add-path::$Env:VS_PATH\$Env:COMPILER_SUBPATH;C:\nasm"
-    - name: Install Rust
+    - name: Install ${{ matrix.toolchain }}
```

Comment:
Is this related?

Reply:
Hmm, I don't think so, but I can investigate on it